src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("The Excavator is doing some work");
```

Comment:
The message 'The Excavator is doing some work' could be more informative by specifying that it is an Excavator type of Machine that is working.